scripts/generate/example.js

```diff
@@ -0,0 +1,23 @@
+const commandLineArgs = require('command-line-args');
+const exec = require('../helpers/bash');
+
+const cmd = commandLineArgs([{
+	name: 'name',
+	defaultOption: true
+}], {
+	stopAtFirstUnknown: true
+});
+
+if (!cmd.name) {
+    console.error("Package name is required.");
+    process.exit(1);
+}
+
+// TODO: remove force flag once https://github.com/angular/angular-cli/issues/11337 is fixed
```

Comment:
i think this one is ok to leave in, but there should still be a corresponding issue created so it isn't forgotten